src/behavior.scm

```diff
@@ -203,6 +203,29 @@
 			(ListLink (Node "--- Glance at new person")))
 	))
 
+;; New recognized person sequence
+(DefineLink
+	(DefinedPredicate "Interacting Sequence for recognized person")
+	(SequentialAnd
+		;(True (DefinedSchema "glance at new person"))
+		(True (Put ; Setting interaction target for "look at person"
+			(DefinedPredicate "Set interaction target")
+			(RandomChoice (Put; FIXME Replace by multiple face tracking.
+				(DefinedSchemaNode "Get recognized face's face id")
+				(DefinedSchema "Get recognized faces")))))
+		(DefinedPredicate "look at person")
+		(True (Put
+			(DefinedPredicate "Greet recognized person")
```

Comment:
OK,  Well, in the various comments around here, I think I sketched how I think it should work. I'll try to provide more detail in the coming weeks, but mostly, I want it to work in a fashion similar to whats in the "chatbot-eva" "self-awareness" blob of ideas, except that "self-awareness" is expanded to "world-awareness" (someone entered the room) and that openpsi makes the final decisions of what  to do (say hello, or maybe stick out her tongue - oh wait, can she do that?)